api/nodes/serializers.py

```diff
@@ -206,7 +207,7 @@ def get_license_details(node, validated_data):
     if ('license_type' not in validated_data and not (license and license.node_license.license_id)):
         raise exceptions.ValidationError(detail='License ID must be provided for a Node License.')
     license_id = license.node_license.license_id if license else None
-    license_year = license.year if license else None
+    license_year = license.year if license else datetime.datetime.now()
```

Comment:
this should be a string of the year. datetime.datetime.now().strftime('%Y') would probably be the canonical approach. str(datetime.datetime.now().year) would also technically work.